website/src/data/marketplace.js

```diff
@@ -175,7 +175,7 @@ export const extensions = [
       "A ruleset which enables verifying whether concrete classes are implemented as specified according to annotations applied to base types.",
     repo: "https://github.com/cph-cachet/detekt-verify-implementation",
     ruleset: "verify-implementation",
-    rules: ["Immutable", "DataClass"],
+    rules: ["DataClass", "Immutable"],
```

Comment:
wrap down for consistency?
```suggestion
    rules: [
      "DataClass",
      "Immutable",
    ],
```

Reply:
This would require reworking the whole file, not just this line. As long as a single line doesn't become too long, I'm fine with this format.